packages/react-native-bridge/common/gutenberg-web-single-block/prevent-autosaves.js

```diff
@@ -3,4 +3,9 @@ window.setTimeout( () => {
 	const settings = window.wp.data.select( 'core/editor' ).getEditorSettings();
 	settings.autosaveInterval = 60 * 60 * 24 * 7; //Let's wait a week for it to autosave.
 	window.wp.data.dispatch( 'core/editor' ).updateEditorSettings( settings );
+
+	const nuxStore = window.wp.data.dispatch( 'automattic/nux' );
+	if ( nuxStore ) {
+		nuxStore.setWpcomNuxStatus( { isNuxEnabled: false } );
```

Comment:
Just ran into this code today. I was curious, why do we reference Atuomattic/WordPress.com specific code here?